Src/VsVimShared/Implementation/OptionPages/KeyboardSettingsControl.xaml.cs

```diff
@@ -134,31 +126,6 @@ private void OnDisableAllVimKeysClick(object sender, RoutedEventArgs e)
             _keyBindingList.ForEach(x => x.HandledByVsVim = false);
         }
 
-        /// <summary>
-        /// Is this an advanced key stroke like CTRL+V which we don't want to automatically 
-        /// convert.  
-        /// </summary>
-        private bool IsAdvanced(KeyStroke keyStroke)

```

Comment:
I've honestly never liked this function and behavior. Always felt like a hack. But I was a bit reluctant to take over literally everything here. Feel like there is a chance of backlash around this but think it's worth trying out.